MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -252,8 +252,14 @@ private async Task TrySaveToFiles(Stream stream, List<string> savePaths)
                 }
                 catch (Exception ex)
                 {
-                    (exs ??= new List<Exception>()).Add(ex);
-                }
+#pragma warning disable CA1508
```

Comment:
Maybe add a note that this is a bug in the analyzer?
Could also add a ref
https://github.com/dotnet/roslyn-analyzers/issues/5160